src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -434,6 +436,8 @@ class AresDNSResolver : public DNSResolver {
           }
         }
       }
+      grpc_pollset_set_del_pollset_set(request->interested_parties_,
```

Comment:
The two parameters are in the opposite order here than they are in `Cancel()` (line 411).

I can never remember which order is correct, but I strongly suspect they need to be the same in both places.